### DISCUSSION TITLE
entitys and items rendering weird when portal in view

### DISCUSSION THREAD

**Discussion by Albinkizi** [iPortalTeam/ImmersivePortalsMod#1612] [Non-Compatibility Issue Report]
### Minecraft version

1.21.1

### Describe the issue

when I have a portal in sight entities and items render wrong like in the air or in blocks and when I don't have a portal in sight they render correctly.
i only had immersive portals as mod and the fabric api in on curseforge fabric.
![2024-08-31_13 45 52](https://github.com/user-attachments/assets/1d3dee53-db64-4184-ae27-3fc13bde4ffa)
![2024-08-31_13 45 58](https://github.com/user-attachments/assets/e14ef026-b9d1-424f-9b17-2192f6c82fea)
they follow my head movment when i saw a portal
sorry my bad English I tried using google translate

**Comment by qouteall** [comment]
Weird, I cannot reproduce this issue.

**Comment by qouteall** [comment]
Made some rendering changes in 6.0.1. Not sure whether it's fixed because I cannot reproduce the issue.

https://github.com/iPortalTeam/ImmersivePortalsMod/releases/tag/v6.0.1-mc1.21.1

**Comment by Albinkizi** [comment]
can not recreate it with update 6.0.1
